addons/zeus/functions/fnc_ui_manageAirway.sqf

```diff
@@ -0,0 +1,118 @@
+#include "script_component.hpp"
+/*
+ * Author: DiGi
+ * Initalizes the "Manage Airways" Zeus module.
+ *
+ * Arguments:
+ * 0: manageAirway controls group <CONTROL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [control] call kat_zeus_fnc_ui_manageAirway
+ *
+ * Public: No
+ */
+
+params ["_control"]; 
+
+private _display = ctrlParent _control;
+private _ctrlButtonOK = _display displayCtrl 1;
+private _logic = GETMVAR(BIS_fnc_initCuratorAttributes_target,objNull);
+
+_control ctrlRemoveAllEventHandlers "setFocus";
+
+
+private _unit = attachedTo _logic;
+
+scopeName "Main";
+private _fnc_error = {
+    params ["_msg"];
+    _display closeDisplay 0;
+    deleteVehicle _logic;
+    [_msg] call FUNC(showMessage);
+    breakOut "Main";
+};
+
+switch (false) do {
+    case !(isNull _unit): {
+        [LSTRING(NothingSelected)] call _fnc_error;
+    };
+    case (_unit isKindOf "CAManBase"): {
+        [LSTRING(OnlyInfantry)] call _fnc_error;
+    };
+};
+
+private _fnc_onUnload = {
+    private _logic = GETMVAR(BIS_fnc_initCuratorAttributes_target,objNull);
+    if (isNull _logic) exitWith {};
+    deleteVehicle _logic;
+};
+
+private _fnc_sliderMove = {
+    params ["_slider","_curSpO2Val"];
+    private _idc = ctrlIDC _slider;
+	private _logic = GETMVAR(BIS_fnc_initCuratorAttributes_target,objNull);
+	private _unit = attachedTo _logic;
+    private _curVal = _unit getVariable ["KAT_medical_airwayStatus", 50];
+    _slider ctrlSetTooltip format ["%1%3 (was %2%3)", round(sliderPosition _slider), round _curVal, "%"];
```

Comment:
Tooltip should be added to the stringtable.xml too

Reply:
Solved